refactor (#325)
KxSystems/docs: docs/wp/data-recovery.md

> In a standard tick system, each kdb+ message will call a function named `upd`. Each process may have a different definition of this function. A TP will publish each time the `upd` function is called (if its timer is not set to batch the data), and an RDB will simply insert the data into the relevant table.

Reviewer: ```suggestion
In a standard tick system, each kdb+ message calls a function named `upd`. Each process may have a different definition of this function. A TP publishes each time the `upd` function is called (if its timer is not set to batch the data), and an RDB inserts the data into the relevant table.
```

### Replaying a tplog

> Recovery of an RDB typically involves simply restarting the process. On startup, an RDB will subscribe to a TP and will receive information about the message count ([`.u.i`](../architecture/tickq.md#variables)) and location of the tplog ([`.u.L`](../architecture/tickq.md#variables)) in return. 
> 
> It will then replay this tplog to recover all the data that has passed through the TP up to that point in the day. The replay is achieved using [`-11!`](../basics/internal.md#-11-streaming-execute), the streaming replay function. 

Reviewer: ```suggestion
Recovery of an RDB involves restarting the process. On startup, an RDB subscribes to a TP and receives the following information:
- message count ([`.u.i`](../architecture/tickq.md#variables)) 
- location of the tplog ([`.u.L`](../architecture/tickq.md#variables)). 

It then replays this tplog to recover all the data that has passed through the TP up to that point in the day. The replay is achieved using [`-11!`](../basics/internal.md#-11-streaming-execute), the streaming replay function. 
```

### `-11!` functionality

> [`-11!`](../basics/internal.md#-11-streaming-execute) is an internal function that will read each message in a tplog in turn, running the function `functioname` on the `(tablename;tabledata)` parameters.

Reviewer: ```suggestion
[`-11!`](../basics/internal.md#-11-streaming-execute) is an internal function that reads each message in a tplog in turn, running the function `functioname` on the `(tablename;tabledata)` parameters.
```

> In this section, we will focus on normal usage of the `-11!` function where the tplog is uncorrupted, and move on to discuss how to use it to recover from a corrupted tplog below in [_Replay errors_](#replay-errors).

Reviewer: ```suggestion
This section focuses on normal usage of the `-11!` function where the tplog is uncorrupted, and then discusses how to use it to recover from a corrupted tplog below in [_Replay errors_](#replay-errors).
```
